tools/accuracy_checker/accuracy_checker/annotation_converters/open_images_converter.py

```diff
@@ -0,0 +1,87 @@
+"""
+Copyright (c) 2018-2021 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from ..config import PathField
+from ..representation import DetectionAnnotation
+from ..utils import read_csv, check_file_existence
+from .format_converter import BaseFormatConverter, ConverterReturn
+
+
+class OpenImagesConverter(BaseFormatConverter):
+    __provider__ = 'open_images'
```

Comment:
dataset support multiple tasks
```suggestion
    __provider__ = 'open_images_detection'
```